File: pset1/cash/cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main(void)
{
    float Change;
        do
        {
            Change = get_float("Change owed: ");
        }
        while (Change<=0);

        int Cents = round(Change*100);

        int Count=0, Quarter=25, Dime=10, Nickel=5, Penny=1;
        while (Cents>=Quarter)
        {
            Count++;
            Cents=Cents-Quarter;
        }
        while (Cents>=Dime)
        {
            Count++;
            Cents=Cents-Dime;
        }
        while (Cents>=Nickel)
        {
            Count++;
            Cents=Cents-Nickel;
        }
        while (Cents>=Penny)
        {
            Count++;
            Cents=Cents-Penny;
        }
        printf("%i\n", Count);
}

File: pset6/cash/cash.py
from cs50 import get_float

change = 0
while change <= 0:
    change = get_float('Change owed: ')

change = round(change*100)
count = 0
quarter, dime, nickel, penny = 25, 10, 5, 1
while change >= quarter:
    count += 1
    change = change - quarter
while change >= dime:
    count += 1
    change = change - dime
while change >= nickel:
    count += 1
    change = change - nickel
while change >= penny:
    count += 1
    change = change - penny

print(count)
